sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/implementation/models/MultipartPart.java

```diff
@@ -16,7 +16,7 @@
     private final List<T> contents = new ArrayList<>();
 
     public MultipartPart(String boundaryPrefix) {
-        this.boundary = boundaryPrefix + "_" + UUID.randomUUID().toString();
+        this.boundary = boundaryPrefix + "_" + UUID.randomUUID();
```

Comment:
Oh cool, as I understand this, Java calls toString() automatically when concatenating strings. Is that correct?

Reply:
That's correct.